angular/test/test-app/src/main.server.ts

```diff
@@ -9,3 +9,5 @@ if (environment.production) {
 export { AppServerModule } from './app/app.server.module';
 export { ngExpressEngine } from '@nguniversal/express-engine';
 export { provideModuleMap } from '@nguniversal/module-map-ngfactory-loader';
+
+export { renderModule, renderModuleFactory } from '@angular/platform-server';
```

Comment:
please add a missing new line at the eof

Reply:
Done :)